stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/viewholder/attachment/MediaAttachmentsViewHolder.kt

```diff
@@ -0,0 +1,41 @@
+package io.getstream.chat.android.ui.message.list.adapter.viewholder.attachment
+
+import com.getstream.sdk.chat.adapter.MessageListItem
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.ui.databinding.StreamUiItemImageAttachmentBinding
+import io.getstream.chat.android.ui.message.list.adapter.MessageListListenerContainer
+import io.getstream.chat.android.ui.message.list.adapter.view.internal.AttachmentClickListener
+import io.getstream.chat.android.ui.message.list.adapter.view.internal.AttachmentLongClickListener
+
+internal class MediaAttachmentsViewHolder(
+    private val binding: StreamUiItemImageAttachmentBinding,
+    container: MessageListListenerContainer?,
+) : AttachmentViewHolder(binding.root) {
+
+    private val mediaAttachmentsView = binding.mediaAttachmentView
+    private var message: Message? = null
```

Comment:
Shouldn't this be `mediaAttachmentsGroupView` both here in the layout? I got a bit confused thinking it was the other view

Reply:
Good call, I updated this.